File: Assets/YesorNo.js
#pragma strict
var Word : UnityEngine.UI.Text;
var Press : GameObject;

function Start () {
	Word.text = "";
	Press.SetActive (true);
}

function yesorno () {
	var number = Random.Range(1,3);
	Press.SetActive (false);
	if (number == 1) {
		Word.text = "Yes!";
	}else if (number == 2) {
		Word.text = "No!";
	}
}

File: Assets/YesorNo.cs
using UnityEngine;
using System.Collections;

[System.Serializable]
public partial class YesorNo : MonoBehaviour
{
    public UnityEngine.UI.Text Word;
    public GameObject Press;
    public virtual void Start()
    {
        this.Word.text = "";
        this.Press.SetActive(true);
    }

    public virtual void yesorno()
    {
        int number = Random.Range(1, 3);
        this.Press.SetActive(false);
        if (number == 1)
        {
            this.Word.text = "Yes!";
        }
        else
        {
            if (number == 2)
            {
                this.Word.text = "No!";
            }
        }
    }

}